pkg/deploy/util/util.go

```diff
@@ -141,14 +141,18 @@ func MakeDeployment(config *deployapi.DeploymentConfig, codec runtime.Codec) (*a
 		selector[k] = v
 	}
 	selector[deployapi.DeploymentConfigLabel] = config.Name
+	selector[deployapi.DeprecatedDeploymentConfigLabel] = config.Name
 	selector[deployapi.DeploymentLabel] = deploymentName
+	selector[deployapi.DeprecatedDeploymentLabel] = deploymentName
```

Comment:
Darn, forgot that all labels from the selector need to exist in labels... Right.